RRemoteConfig/Environment.swift

```diff
@@ -0,0 +1,27 @@
+internal protocol EnvironmentSetupProtocol {
+    func value(for key: String) -> String?
+}
+
+extension Bundle: EnvironmentSetupProtocol {
+    func value(for key: String) -> String? {
+        return self.object(forInfoDictionaryKey: key) as? String
+    }
+}
+
+internal class Environment {
+    let bundle: EnvironmentSetupProtocol
+    var url: URL? {
+        guard let appId = bundle.value(for: "RASApplicationIdentifier"), let endpointUrlString = bundle.value(for: "RRCConfigAPIEndpoint") else {
+            print("Ensure RASApplicationIdentifier and RRCConfigAPIEndpoint values in plist are valid")
+            return nil
+        }
+        return URL(string: "\(endpointUrlString)/\(appId)/config")
```

Comment:
I think `app/` should be added within this concatenation and not be part of `RRCConfigAPIEndpoint`. For example, this will match the format used by the base URL in performance tracking. Also that will make the API version be the last path segment of the base URL.

Also, it kind of feels like the `config` path should be appended within the ConfigFetcher. Though we most likely won't have any more endpoints, so maybe it doesn't matter. What do you think?

Reply:
I did it like this cos there is only 1 endpoint but I'll change it to add `app/` here.

Reply:
I guess it doesn't matter for now because there will most likely be only one endpoint. We can always change it if we need to add another one.